pkg/operator/ceph/object/user.go

```diff
@@ -185,6 +198,11 @@ func DeleteUser(c *Context, id string, opts ...string) (string, error) {
 		if code, ok := exec.ExitStatus(err); ok && code == int(syscall.ENOENT) {
 			return result, nil
 		}
+
+		res, er := ListUserBucket(c, id)
```

Comment:
```suggestion
		res, err := ListUserBucket(c, id)
```

Reply:
Doing this intentionally as I need `err` from line no. 195